Documentation/ApiOverview/FormEngine/Rendering/_ImportDataControl.php

```diff
@@ -18,6 +18,8 @@ public function render(): array
                 'class' => 'importData ',
                 'data-id' => $this->data['databaseRow']['somefield'],
             ],
+            'javaScriptModules' => ['@my_vendor/my_extension/import-data.js'],
```

Comment:
As far as I understand: now in v12 both JS files are included: The ES6 and RequireJS one.

So, when using the javaScriptModules in combination with requireJsModules, we need a version check.

Reply:
As far as I understand this we are just providing information here so we can provide both?

Reply:
Okay,  looked up the code, resulting in this example to add both: the ES6 and RequireJS JavaScripts:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/backend/Classes/Form/FormResultCompiler.php#L207-L213

Reply:
done